app/validators/claim/litigator_claim_sub_model_validator.rb

```diff
@@ -4,13 +4,9 @@ def has_one_association_names_for_steps
       case_details: [],
       defendants: [],
       offence_details: [],
-      fees: %i[
-        graduated_fee
-        fixed_fee
-        warrant_fee
-        assessment
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2252/files#r170615857)